### DISCUSSION TITLE
Script idea: "time machine" for likes, inbox, etc

### DISCUSSION THREAD

**Discussion by marcustyphoon** [AprilSylph/XKit-Rewritten#440] [Suggestion Box]
Tumblr has quietly reimplemented some form of pagination or ability to skip backwards on multiple timeline types (the totality of which I have not kept track... of*). It would be nice to have a script that adds an element with controls to jump back to offset x, on pages that use a numerical offset like the inbox, or to jump back to a certain date, on pages that use a unix timestamp.

Not actually sure where this would go (April, I know you prefer to avoid adding elements to the page that won't have a place to go on the mobile website). A sidebar section makes sense besides the aforementioned limit; putting an element in the timeline itself preceding the first timeline object would be one way of solving this.

_*I have no idea how one would actually make this work, grammatically, but I'm committed now_

**Comment by AprilSylph** [comment]
I think it would be best as preceding the first timeline object. The sidebar approach would work (sidebar items get put in the drawer on the mobile site now! see a18cdca583f740c420eb1ff1732ed86b1415e60f) but it would be kinda weird to have the time machine always visible, and I think the feature could enjoy more horizontal space to work with.
